auto_backup/data/backup_data.yml

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
```

Comment:
Awesome well that just made me incredibly happy - I hate the YML tests. I'll move this to Python